src/Microsoft.DotNet.Wpf/src/WindowsBase/System/IO/Packaging/PackageDigitalSignatureManager.cs

```diff
@@ -470,7 +470,7 @@ public PackageDigitalSignature Sign(IEnumerable<Uri> parts, X509Certificate cert
             VerifySignArguments(parts, certificate, relationshipSelectors, signatureId, signatureObjects, objectReferences);
 
             // substitute default id if none given
-            if ((signatureId == null) || (signatureId == String.Empty))
+            if (String.IsNullOrEmpty(signatureId))
```

Comment:
It has to be `string` and not `String`
```suggestion
            if (string.IsNullOrEmpty(signatureId))
```

Applies universally.